rsyslog/utils/logs

```diff
@@ -42,10 +42,12 @@ done
 
 if [ $cmd = "cat" ] ; then
     if [ "$include_logrotation" = "true" ] ; then
-        echo "===== rsyslog logs ====="
-        for log_file in $( ls $log_dir/rsyslog.* | sort -Vr ); do
-            cat $log_file
-        done
+        if ls $pod_dir/rsyslog.* > /dev/null 2>&1 ; then
```

Comment:
this is to fix the bz?

Reply:
Yes, I asked QE to open a bug by mistake...  I thought the bug was already in the master branch, but it was only in this PR.  I added the logs utility improvement which was needed for debugging.